api/_examples/cloud-accouts/main.go

```diff
@@ -78,11 +80,33 @@ func main() {
 		log.Fatal(err)
 	}
 
-	response, err := orgLwClient.V2.CloudAccounts.Create(myCloudAccount)
+	awsCtSqsResponse, err := orgLwClient.V2.CloudAccounts.Create(awsCtSqsCloudAccount)
```

Comment:
Can we move these to 2 subfolders inside cloudaccounts. I think it's better if we separate the examples.

Eg.
```
 _examples/cloud_accounts/eks_audit/main.go
_examples/cloud_accounts/aws_cloudtrail/main.go
```

Reply:
Yeah makes sense. Was thinking that they should probably be separate, but there wasn't any precedent in this area, so left it as a flat file.